lib/fathead/mdn_css/parse.pl

```diff
@@ -215,27 +198,44 @@ sub clean_string {
     return $clean;
 }
 
+sub build_initial_value {
+    my ($content) = @_;
+    my $initial_value = "<p><b>Initial Value: </b><em>$content</em></p>";
+    return $initial_value;
+}
+
 sub build_abstract {
-    my ( $description, $code ) = @_;
+    my ( $description, $code, $initial_value ) = @_;
     say "NO DESCRIPTION!" if $description eq "";
+    $description = trim($description);
+    $description =~ s/\r?\n+/\\n/g;
+    $initial_value =~ s/\r?\n+/\\n/g if $initial_value;
+    $code = clean_code($code) if $code;
     my $out;
-    $out .= "<p>$description</p>" if $description;
-    $out .= "<br><pre><code>$code</code></pre>" if $code;
+    $out .= "<p>$description</p>"           if $description;
+    $out .= "$initial_value"                if $initial_value;
```

Comment:
Perhaps we should wrapped this in `<p>` here instead of inside `build_initial_value`?